.github/workflows/e2e.yml

```diff
@@ -46,8 +46,9 @@ jobs:
           kurtosis engine start
           kurtosis run --enclave op-devnet github.com/ethpandaops/optimism-package --args-file ./etc/kurtosis.yaml
           ENCLAVE_ID=$(curl http://127.0.0.1:9779/api/enclaves | jq --raw-output 'keys[0]')
-          SEQUENCER_EL_PORT=$(curl "http://127.0.0.1:9779/api/enclaves/$ENCLAVE_ID/services" | jq '."op-el-1-op-reth-op-node-op-kurtosis".public_ports.rpc.number')
-          REPLICA_EL_PORT=$(curl "http://127.0.0.1:9779/api/enclaves/$ENCLAVE_ID/services" | jq '."op-el-2-op-reth-op-node-op-kurtosis".public_ports.rpc.number')
+          SEQUENCER_EL_PORT=$(curl "http://127.0.0.1:9779/api/enclaves/$ENCLAVE_ID/services" | jq '."op-el-41144114-1-op-reth-op-node-op-kurtosis".public_ports.rpc.number')
+          curl "http://127.0.0.1:9779/api/enclaves/$ENCLAVE_ID/services" | jq '.'
+          REPLICA_EL_PORT=$(curl "http://127.0.0.1:9779/api/enclaves/$ENCLAVE_ID/services" | jq '."op-el-41144114-2-op-reth-op-node-op-kurtosis".public_ports.rpc.number')
```

Comment:
curios, where do we get this identifier from

similar thing here

https://github.com/paradigmxyz/reth/blob/eda2b09132ea761485326c9c810a837f126dfdd7/.github/workflows/kurtosis-op.yml#L67-L68

and not obvious where this is coming from

maybe @fgimenez knows?